<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>labelTool</class>
 <widget class="QMainWindow" name="labelTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>labelTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>823</width>
         <height>592</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFiles"/>
    <addaction name="separator"/>
    <addaction name="actionOpenDir"/>
    <addaction name="separator"/>
    <addaction name="actionSetAnnotationDir"/>
    <addaction name="separator"/>
    <addaction name="actionLoadLabels"/>
    <addaction name="separator"/>
    <addaction name="actionLoadDatasets"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Dataset"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="separator"/>
    <addaction name="actionZoomOut"/>
    <addaction name="separator"/>
    <addaction name="actionFitWindow"/>
    <addaction name="separator"/>
    <addaction name="actionNextImage"/>
    <addaction name="separator"/>
    <addaction name="actionPreviousImage"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QDockWidget" name="AnnotationDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>138</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="windowTitle">
    <string>Annotation List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QListWidget" name="AnnotationList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="LabelDock">
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>138</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="windowTitle">
    <string>Label List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QListWidget" name="LabelList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="FileDock">
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>138</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>File List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <widget class="QListWidget" name="FileList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>161</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="mousePosLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionFitWindow"/>
   <addaction name="actionOpenFiles"/>
   <addaction name="actionOpenDir"/>
   <addaction name="actionPreviousImage"/>
   <addaction name="actionNextImage"/>
   <addaction name="actionSetAnnotationDir"/>
   <addaction name="actionLoadLabels"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionOpenFiles">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/file.png</normaloff>:/icons/file.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpenDir">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Dir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/zoom-in.png</normaloff>:/icons/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/zoom-out.png</normaloff>:/icons/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFitWindow">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/zoom.png</normaloff>:/icons/zoom.png</iconset>
   </property>
   <property name="text">
    <string>Fit Window</string>
   </property>
  </action>
  <action name="actionNextImage">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
   </property>
   <property name="text">
    <string>Next Image</string>
   </property>
  </action>
  <action name="actionPreviousImage">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/prev.png</normaloff>:/icons/prev.png</iconset>
   </property>
   <property name="text">
    <string>Previous Image</string>
   </property>
  </action>
  <action name="actionSetAnnotationDir">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/pinned.svg</normaloff>:/icons/pinned.svg</iconset>
   </property>
   <property name="text">
    <string>Set Annotation Save Dir</string>
   </property>
  </action>
  <action name="actionLoadLabels">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/pinned_dirty.svg</normaloff>:/icons/pinned_dirty.svg</iconset>
   </property>
   <property name="text">
    <string>Load Labels</string>
   </property>
  </action>
  <action name="actionLoadDatasets">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/import.png</normaloff>:/icons/import.png</iconset>
   </property>
   <property name="text">
    <string>Load Dataset</string>
   </property>
  </action>
  <action name="actionSave_Dataset">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/export.png</normaloff>:/icons/export.png</iconset>
   </property>
   <property name="text">
    <string>Save Dataset</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/quit.png</normaloff>:/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/save.svg</normaloff>:/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
